pkg/infra/kubernetes/helm_chart.go

```diff
@@ -14,23 +14,49 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+type HelmFetchOpts struct {
+	Repo string
+}
+
 type HelmChart struct {
-	Name           string
-	ValuesFiles    []string
-	ExecutionUnits []*HelmExecUnit
-	Directory      string
-	Files          []core.File
-	Values         []Value
-	AnnotationKeys []core.AnnotationKey
+	Name               string
+	Chart              string
+	ValuesFiles        []string
+	ExecutionUnits     []*HelmExecUnit
+	Directory          string
+	Files              []core.File
+	Values             []Value `render:"document"`
```

Comment:
im not sure that this will really be rendered correctly because values is a map. This i think is the trickiest part that we have to deal with because our Values today were internally mapped so they placed nicely with how we were doing things in pulumi. But think of an external chart, how do we propose setting values here?

Reply:
I think that's the next piece for me to look into. I haven't gotten to that piece yet.